<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CK2iLogWindow</class>
 <widget class="QMainWindow" name="CK2iLogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CK2iLogWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="log_display">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>281</width>
      <height>361</height>
     </rect>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="DecisionBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>561</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Decisions</string>
    </property>
    <widget class="QPushButton" name="PickDecisionSourceBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Pick Decisions Directory</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DecisionLogStartBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Convert Decisions</string>
     </property>
    </widget>
    <widget class="QLabel" name="DecisionDirInfoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>521</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="DecisionLogProgressBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>521</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="CurrentDecisionLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>521</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="EventBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>561</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Events</string>
    </property>
    <widget class="QPushButton" name="PickEventSourceBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Pick Events Directory</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EventLogStartBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Convert Events</string>
     </property>
    </widget>
    <widget class="QLabel" name="EventDirInfoLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>521</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="EventLogProgressBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>521</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="CurrentEventLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>491</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="InsertInTriggersCB">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert log in Trigger Blocks</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug Log Output:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ScanLogfileBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan Logfile</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
